tests/TaskLineRenderer.test.ts

```diff
@@ -296,6 +302,18 @@ describe('task line rendering', () => {
         },
     );
 
+    it('"hide tags" should not hide Global Filter when it is a tag', async () => {
```

Comment:
I think it should hide the global filter if it is a tag, and `hide tags` was used. Why do you think not?

Reply:
Indeed, I misinterpreted your message, thanks fro your thorough review =) I suggest to resolve this conversation if the new test is ok